files/image_config/sysctl/sysctl-net.conf

```diff
@@ -14,7 +14,6 @@ net.ipv4.conf.all.arp_accept=0
 net.ipv4.conf.all.arp_announce=1
 net.ipv4.conf.all.arp_filter=0
 net.ipv4.conf.all.arp_notify=1
-net.ipv4.conf.all.arp_ignore=2
```

Comment:
Do not change default values that break backward compatibility.

Reply:
According to linux documentation, https://github.com/torvalds/linux/blob/master/Documentation/networking/ip-sysctl.rst  
`The max value from conf/{all,interface}/arp_ignore is used when ARP request is received on the {interface}`

arp_ignore will take the max value, therefore all interfaces will use the global value. 
If this setting could be changed to be controlled via the interface, it would provide users with more flexibility.
Therefore, remove the global value and change the default arp_ignore setting of 2 to be configured per interface.
Related PR: [sonic-net/sonic-swss#3122](https://github.com/sonic-net/sonic-swss/pull/3122)